Nav.php

```diff
@@ -116,7 +126,7 @@ public function init()
         if ($this->dropDownCaret === null) {
             $this->dropDownCaret = Html::tag('b', '', ['class' => 'caret']);
         }
-        Html::addCssClass($this->options, 'nav');
+        Html::addCssClass($this->options, $this->menuCssClass);
```

Comment:
Such construction does not seem right: we have a `$options`, which allows specification of the 'class' and `$menuCssClass` for the specification of the class.
It seems either `Html::addCssClass()` usage should be abandonded or `menuCssClass` not introduced.